add reprewards information (#76)
botlabs-gg/yagpdb-docs-v2: content/docs/fun/reputation.md

Users with this role cannot participate in the point system which they can be given/taken away points from. This also
overrides the _Required role to receive/being taken away points from_.

> ## Reputation roles

Reviewer: Title Case maybe? Though the header on the control panel isn't, so I'm not sure.

Author: yea, I referenced the page specifically for this one and figured if i made it different I'd get a comment about not matching, but I agree, it felt wrong here.

Reviewer: let's keep it as-is

You can set up role rewards for reaching certain reputation amounts. Up to 5 rewards on free, 25 for premium. This system
is basic and offers little customization. For more advanced setups, custom commands are recommended.

![Overview of Reputation Role Rewards](overview_reuptationRewards.png)

### Create new reputation role

Select a rep point threshold and role to assign when a user receives that many points.

### Existing reputation roles

> This will show all existing reputation roles on your guild

Reviewer: ```suggestion
A listing of all your existing reputation roles with their respective threshold.
```
I'd like this to be a full sentence.
